src/openapi/streaming/connection/connection.js

```diff
@@ -2,7 +2,7 @@ import log from './../../../log';
 import WebsocketTransport from './transport/websocket-transport';
 import SignalrTransport from './transport/signalr-transport';
 
-const LOG_AREA = 'PlainWebSocketsTransport';
+const LOG_AREA = 'Connection';

```

Comment:
Also add more info in transport failed log in onTransportFail
i.e. which transport failed

Reply:
Done